### PR TITLE
Project Settings 'restart' message put focus on editor rather than project itself

### LINKED ISSUE #39613: Misleading warning about vsync requiring an editor restart in the Project Settings
**Godot version:**

3.2.2 rc1

**Issue description:**

When you toggle `vsync` in Project Settings, you get a warning that you need to restart editor for the change to take effect. This is untrue. It takes effect immediately, at least in the project. If the warning is about vsync in the editor, it should state that you need to restart editor to affect the editor. Right now it sounds like it won't have any effect until restart.

### PR DESCRIPTION
**Pull request by dreamsComeTrue**
Fixes: #39613

### CODE REVIEW

File: editor/project_settings_editor.cpp [hunk 1]
```diff
@@ -1865,7 +1865,7 @@ ProjectSettingsEditor::ProjectSettingsEditor(EditorData *p_data) {
 	restart_icon->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
 	restart_hb->add_child(restart_icon);
 	restart_label = memnew(Label);
-	restart_label->set_text(TTR("The editor must be restarted for changes to take effect."));
+	restart_label->set_text(TTR("Changed settings will be applied to the editor after restart."));
```
Comment (Calinou) [inline editor/project_settings_editor.cpp:1868]:
```suggestion
	restart_label->set_text(TTR("Changed settings will be applied to the editor after restarting."));
```

**Review by Calinou — CHANGES_REQUESTED** [review]

File: editor/project_settings_editor.cpp (hunk 1 above)
Comment (dreamsComeTrue) [reply editor/project_settings_editor.cpp]: Done!

**Review by aaronfranke — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Not cherry-picking for 3.2.2 as it would break translations, and IMO the original string wasn't bad enough to justify it in the stable branch (but it's fine for 4.0, translations will have time to be updated).

**Pull request merged**